src/models/ProjectNotification.ts

```diff
@@ -56,53 +56,39 @@ export class ProjectNotification {
             .getData();
 
         const { displayName: user, username } = this.currentUser.data;
+        const subject = i18n.t("{{username}} is requesting a data review", { username });
+        const users = res.userRoles.flatMap(userRole => userRole.users);
+        const dataSet = res.dataSets[0];
 
-        const subject = i18n.t("{{username}} is requesting for data review", { username });
-        const pageLink = window.location.href;
-        const dataSetType = pageLink.includes("actual") ? "ACTUAL" : "TARGET";
-        const userAccessEmails = res.userRoles[0].users
+        const userAccessEmails = users
```

Comment:
Note how you get the users from the roles twice, it can be stored in a local variable. Same for the dataSet.